imports/blocks/add-schedule/index.js

```diff
@@ -1,42 +1,55 @@
-import { Component, PropTypes } from "react";
+// @flow
+
+import serverWatch from "meteor/bjwiley2:server-watch";
+import { Component } from "react";
 import { connect } from "react-redux";
 import moment from "moment";
 import createContainer from "../meteor/react-meteor-data";
-
 import { give as giveActions } from "../../store";
 import Offline from "../../components/status/Offline";
-
 import Layout from "./Layout";
 
-// We only care about the give state
-const map = (state) => ({ give: state.give });
+type ICartContainer = {
+    accounts: Object,
+    addTransactions: Function,
+    alive: boolean,
+    clearAllSchedulesExcept: Function,
+    clearSchedules: Function,
+    clearTransactions: Function,
+    existing: Object,
+    onClick: Function,
+    removeSchedule: Function,
+    saveSchedule: Function,
+    setTransactionType: Function,
+    text: string,
+};
+
+type IState = {
+  amount: ?number,
+  frequency: ?string,
+  fundId: ?number,
+  fundLabel: ?string,
+  startDate: ?string,
+};
+
+// XXX move this to a global constructs file?
```

Comment:
I think we may even be able to just pull this from rock directly using `graphql` if we want (future addition?)